src/services/AuthServices.js

```diff
@@ -6,23 +6,22 @@ export const setCurrentUser = currentUser => {
   api.setHeader('Authorization', currentUser.sessionToken);
   LocalStorageService.setSessionToken(currentUser.sessionToken);
 };
+
 export const getCurrentUser = () => {
   const currentSessionToken = LocalStorageService.getSessionToken();
-
   if (currentSessionToken) {
     api.setHeader('Authorization', currentSessionToken);
-
     return true;
   }
-
   return false;
 };
+
 export const removeCurrentUser = () => LocalStorageService.removeSessionToken();
 
-export const login = () =>
-  // TODO: Implement call to authentication API here
-  new Promise(resolve => {
-    setTimeout(() => {
-      resolve({ ok: true, data: { sessionToken: 'token' } });
-    }, 750); // eslint-disable-line no-magic-numbers
+export const loginToGithub = code => {
+  const authApi = api.setBaseURL(`${process.env.REACT_APP_AUTH_BASE_URL}?code=${code}`);
```

Comment:
Also, you can create another api instance with another base url.
I'm not sure if this `setBaseURL` is persistence between calls, but I'd be careful with that.

Reply:
Sii!! you're right...
I was reading the docu 🙈. And in fact it is necessary to create a new instance!
Thaanks.